packages/dashboard/src/components/lifts-app.tsx

```diff
@@ -25,62 +38,108 @@ const LiftCard = ({ children, ...otherProps }: LiftCardProps) => {
+        <ArrowDownwardIcon
+          sx={currMotion === 'Down' ? motionArrowActiveStyle : motionArrowIdleStyle}
+        />
+      </TableCell>
+      <TableCell align="right" sx={{ marginRight: 1, padding: 0, paddingRight: 1 }}>
+        <LiftControls
+          availableLevels={otherProps.lift.levels}
+          currentLevel={liftState?.current_floor}
+          onRequestSubmit={(_ev, doorState, requestType, destination) =>
+            rmf?.liftsApi.postLiftRequestLiftsLiftNameRequestPost(otherProps.lift.name, {
+              destination,
+              door_mode: doorState,
+              request_type: requestType,
+            })
+          }
+        />
+      </TableCell>
+    </TableRow>
   );
 };
 
 export const LiftsApp = createMicroApp('Lifts', () => {
   const rmf = React.useContext(RmfAppContext);
   const [buildingMap, setBuildingMap] = React.useState<BuildingMap | null>(null);
-  const [liftStates, setLiftStates] = React.useState<Record<string, LiftState>>({});
 
   React.useEffect(() => {
     if (!rmf) {
       return;
     }
     const sub = rmf.buildingMapObs.subscribe((newMap) => {
-      for (const lift of newMap.lifts) {
-        rmf.getLiftStateObs(lift.name).subscribe((state) => {
-          setLiftStates((prev) => ({ ...prev, [lift.name]: state }));
-        });
-      }
       setBuildingMap(newMap);
     });
     return () => sub.unsubscribe();
   }, [rmf]);
 
   return (
-    <Grid container>
-      {buildingMap &&
-        buildingMap.lifts.map((lift) => {
-          const liftState: LiftState | undefined = liftStates[lift.name];
-          return (
-            <LiftCard key={lift.name} name={lift.name} sx={{ width: 200 }}>
-              <CardActions sx={{ justifyContent: 'center' }}>
-                <LiftControls
-                  availableLevels={lift.levels}
-                  currentLevel={liftState?.current_floor}
-                  onRequestSubmit={(_ev, doorState, requestType, destination) =>
-                    rmf?.liftsApi.postLiftRequestLiftsLiftNameRequestPost(lift.name, {
-                      destination,
-                      door_mode: doorState,
-                      request_type: requestType,
-                    })
-                  }
-                />
-              </CardActions>
-            </LiftCard>
-          );
-        })}
-    </Grid>
+    <Table stickyHeader size="small" aria-label="door-table" style={{ tableLayout: 'fixed' }}>
+      <TableHead>
+        <TableRow>
+          <TableCell>Name</TableCell>
+          <TableCell>Current floor</TableCell>
+          <TableCell>Destination floor</TableCell>
+          <TableCell>Door state</TableCell>
+          <TableCell></TableCell>
+        </TableRow>
+      </TableHead>
+      <TableBody>
+        {buildingMap &&
+          buildingMap.lifts.map((lift) => {
+            return <LiftCard key={lift.name} name={lift.name} lift={lift} sx={{ width: 200 }} />;
```

Comment:
Ah I see what's happening here. Since we are no longer using the `LiftCard` from `react-components`, and just using `LiftCardProp` with omitted fields, I would suggest we set up something new with the proper names, like `LiftTableRow` and `LiftTableRowProp`, and we should be able to stop using `LiftCard` altogether.

We can keep the implementation of `LiftTableRow` in `lifts-app` since it is tied to the implementation of the app which is a table.